Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -264,7 +264,7 @@ public struct RNN<Cell: RNNCell>: Layer {
 
     @differentiable(wrt: (self, inputs))
     public func callAsFunction(_ inputs: [Cell.TimeStepInput]) -> [Cell.TimeStepOutput] {
-        return self(inputs, initialState: cell.zeroState.withoutDerivative())
+        return self(inputs, initialState: Swift.withoutDerivative(at: cell.zeroState))
```

Comment:
You should be able to drop `Swift.` when a new toolchain lands. But let's keep it so that CI won't be broken.

```suggestion
        return self(inputs, initialState: withoutDerivative(at: cell.zeroState))
```